malloc/malloc.c

```diff
@@ -1,8 +1,23 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <time.h>
-#include "lib.h"
 
+#include <string.h>
+
+int string_length(char *s)
+{
+  return strlen(s);
```

Comment:
jajajaaa nice job finding this! but try doing it _without_ using premade functions. that's where you get practice with pointers